.eslintignore

```diff
@@ -3,3 +3,5 @@ coverage
 public/js/components
 public/js/vendors
 public/js/demo.js
+public/js/consumptionPrefSort_likely.js
```

Comment:
File names should be `consumption-preferences-likely-soft.js` and  `consumption-preferences-unlikely-soft.js`